indico/modules/events/registration/templates/management/regform_actions/not_started.html

```diff
@@ -29,6 +29,16 @@
             {% endif %}
         </a>
     </div>
+    {% if regform.start_dt %}
```

Comment:
`is_scheduled` might make more sense here (even though it's technically the same)